docs/auth0_configure_application.md

```diff
@@ -2,24 +2,8 @@
 
 The Dock Web3 ID integration allows users to privately sign in to your app using [Decentralized Identifiers (DIDs)](https://docs.api.dock.io/#dids) & [Verifiable Credentials (VCs)](https://www.dock.io/) via scanning QR codes with their Dock Wallet apps. This step-by-step guide will show you how to integrate Dock Web3 ID into your authentication flow easily.
 
-## Under the Hood
```

Comment:
I understand why you moved this, but this document will also be the source of the `Installation` page on the Auth0 marketplace so we need to keep the context for there.

Reply:
i think that we can just copy/paste from the initial readme too when we come to adding that document as im sure theres more context (like the intro paragraph) that would be useful also